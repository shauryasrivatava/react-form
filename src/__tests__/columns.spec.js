import renderer from 'react-test-renderer';
import Column from '../components/Column';
import React from 'react';
import { shallow } from 'enzyme';
import enzymeConfig from '../../enzymeConfig'

describe('<Column />', function() {
    
    it('should render correctly', function() {
        const component = renderer.create(<Column/>);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('should have the correct class name', function () {

        const component = shallow(<Column className="col-md-6" />);
        expect(component.find('div').hasClass('col-md-6')).toBe(true);
    })
})