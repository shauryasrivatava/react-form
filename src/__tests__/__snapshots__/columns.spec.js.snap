// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Column /> should render correctly 1`] = `
<div
  className="col-md-6 offset-md-3"
/>
`;
